### ISSUE TITLE
`includePaths` no longer loading from scss-config.json 

### ISSUE THREAD

**Issue by moberegger** [Meteor-Community-Packages/meteor-scss#306]
- [ ] Feature request
- [x] Bug report
- [ ] Question

`meteor` version: 2.2

`fourseven:scss` version: 4.14.1

[This commit](https://github.com/Meteor-Community-Packages/meteor-scss/commit/e447856af288e58c574af4b67c817906bc626cee) has introduced a bug that will make the `includePaths` from scss-config.json no longer load.

If you look at this logic 
```javascript
function _prepareNodeSassOptions() {
  const config = _loadConfigurationFile();
  if (typeof _includePaths === 'undefined' && config.includePaths) {
    _loadIncludePaths(config);
  }
}
```
it won't set `_includePaths` if it is not undefined; since it has been initialized to `[]`, it will never be undefined, and thus will never get set to the paths from the configuration.

I don't know why this guard is here... it doesn't actually save on anything, because the configuration gets loaded anyways which is actually the expensive part. In fact, the configuration gets loaded every time an attempt is made to load a file this way, which is wasteful. If anything, the guard should be checking if the configuration has already been loaded to prevent that from happening more than once.

As a quick fix, this can probably just be changed to

```javascript
function _prepareNodeSassOptions() {
  const config = _loadConfigurationFile();
  if (config.includePaths) {
    _loadIncludePaths(config);
  }
}
```
There is no harm at all in setting the includePath again, since it should just end up being the same value.

**Comment by moberegger** [comment]
I think we can also get some easy performance wins here by only loading the configuration file once. Right now, it looks like it will do it multiple times per build. For example, if I have a config like 
```json
{
  "includePaths": [
    "{}/node_modules/bootstrap/scss"
  ]
}
```
it looks like it will try to load the config anytime I try to `@import` something from bootstrap, which can get quite expensive.

**Comment by moberegger** [comment]
I have changes locally that fix all of the issues I have mentioned above. I can submit a PR for it soon.

**Comment by sebakerckhof** [comment]
Yeah, my bad, I merged it into an a wrong/incomplete branch locally.

**Comment by StorytellerCZ** [comment]
Fixed in 4.14.2
